recipes/pimcore.php

```diff
@@ -12,10 +12,85 @@
 
 namespace Deployer;
 
+
+
 task('deploy:pimcore:install-classes', function() {
     run('{{bin/php}} {{bin/console}} deployment:classes-rebuild -c');
 });
 
 task('deploy:pimcore:migrate', function() {
     run('{{bin/php}} {{bin/console}} pimcore:migrations:migrate');
+});
+
+desc('Creating symlinks for shared files and dirs');
+task('deploy:shared', function () {
```

Comment:
can we rename that to ```deploy:pimcore:shared:config``` and introduce another configuration ```pimcore:shared_configurations``` to configure them? I don't think it as a good idea to have them static and overwrite a default function of deployer.